csrankings-s.csv

```diff
@@ -802,6 +802,7 @@ Shengyuan Wang,Tsinghua University,http://www.tsinghua.edu.cn/publish/csen/4623/
 Sheqin Dong,Tsinghua University,http://www.tsinghua.edu.cn/publish/csen/4623/2010/20101223231437873460518/20101223231437873460518_.html,VxBX9CgAAAAJ
 Sherah Kurnia,University of Melbourne,https://findanexpert.unimelb.edu.au/display/person6535,vasVu8sAAAAJ
 Sherenaz W. Al Haj Baddar,University of Jordan,http://computer.ju.edu.jo/Lists/FacultyAcademicStaff/Disp_Staff.aspx?ID=18&Dept=Computer%20Science,tdIm-9oAAAAJ
+Sherief Reda,Brown University,https://scale.engin.brown.edu/pages/sreda.html,p69MQp4AAAAJ

```

Comment:
CE != CS. Please indicate affiliation with the CS department or how they meet the inclusion criteria.

Reply:
Prof. Sherief Reda has an appointment in the CS department: https://cs.brown.edu/people/faculty/#:~:text=Sherief%20Reda-,Professor%20of%20Engineering%20and%20Computer%20Science,-Computer%20Systems%2C%20Computer. He has 10+ papers in the computer systems field.